web_interface/blueprints/api_v3.py

```
@@ -1213,24 +1228,17 @@ def start_on_demand_display():
-                    'message': f'Plugin {resolved_plugin} is disabled in configuration'
-                }), 400
-
-        # Check if display service is running (or will be started)
-        service_status = _get_display_service_status()
-        if not service_status.get('active') and not start_service:
-            return jsonify({
-                'status': 'error',
-                'message': 'Display service is not running. Please start the display service or enable "Start Service" option.',
-                'service_status': service_status
-            }), 400
+                logger.info("On-demand request for disabled plugin '%s' - will be temporarily enabled", resolved_plugin)
```

Ruff (0.14.10)

Undefined name `logger`

(F821)